std.ddoc

```diff
@@ -10,9 +10,25 @@ $(SCRIPT jQuery(document).ready(listanchors);)
 LAYOUT_TITLE=$(H1 $(D $(TITLE)))
 _=
 
+_= a NOTE is just something you want to draw attention to. e.g. "this function is Windows only"
+NOTE = <div class="note">$0</div>
+_= a WARNING is something that users often get wrong, but isn't necessarily fatal, like the difference between byLine and byLineCopy
+WARNING = <div class="warning">$0</div>
+_= PITFALL is something that if you get it wrong, your program will crash or other such serious consequences, like passing a GC reference to a C function to store
+PITFALL = <div class="pitfall">$0</div>
+_= a TIP is just a cool fact about using this item
+TIP = <div class="tip">$0</div>
+_= SIDEBAR is something for users who are interested into diving much deeper than they have to to understand the function. It should just contain a short sentence and probably a link to learn more.
+SIDEBAR = <div class="sidebar">$0</div>
+
+_=
+
 CXREF = <a href="core_$1.html#$2">$(D core.$1.$2)</a>
 ECXREF = <a href="etc_c_$1.html#$2">$(D etc.c.$1.$2)</a>
 LREF = <a href="#$1">$(D $1)</a>
+MREF_HELPER=_$1$(MREF_HELPER $+)
+MREF_HELPER2=.$1$(MREF_HELPER2 $+)
```

Comment:
I'm only talking about the generic macros `XXX_PREFIXED`. They don't prescribe absolute vs. relative linking etc.